app/controllers/concerns/reports_actions.rb

```diff
@@ -56,7 +56,7 @@ def ransack_params
   end
 
   def report_format
-    params[:report_format]
+    params[:report_format].presence || "html"
```

Comment:
I'm intrigued if this `.presence` is necessary?
The example in the docs refers to `params` also, but as far as I can tell a non-existent param returns `nil` anyway. IE this works:

```
Hash.new[:format] || "html"
=> "html"
 ActionController::Parameters.new[:format] || "html"
=> "html"
```

so I imagine this is functionally equivalent:
```suggestion
    params[:report_format] || "html"
```

Reply:
When you hit the submit button on the report with the "On Screen" option then `params[:report_format]` will be a blank string.
```rb
params[:report_format] || "html"
=> ""

params[:report_format].presence || "html"
=> "html"
```

Reply:
Oh, I need to revise my Ruby knowledge! I thought values like `""` and `0` would be falsy, but I was wrong:

> In Ruby, only `false` and `nil` are “falsey,” everything else is “truthy.”

https://www.freecodecamp.org/news/data-types-in-ruby-true-false-and-nil-explained-with-examples/#:~:text=In%20Ruby%2C%20only%20false%20and%20nil%20are%20%E2%80%9Cfalsey%2C%E2%80%9D%20everything%20else%20is%20%E2%80%9Ctruthy.%E2%80%9D